### PR TITLE
release: automate release yamls

### PR DESCRIPTION
**Pull request by astoycos**
Add a script which generates ALL of the release yamls needed to get bpfd up and running easily on a kind cluster. This includes the CRD + opeator install as well as the example's digests.

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -101,14 +101,12 @@ For a **PATCH** release:
...
 - The Release will be automatically created, after that is complete do the following:
-  - run `make build-crd-install-yaml` and attach the crd install yaml for the version to the release.
-  - manually build and attach a `bpfd-operator-install-vx.x.x.yaml` to provide an easy to use install
-    yaml for the bpfd-operator.
-  - manually build and attach the following files:
-    - `go-xdp-counter-install-vx.x.x.yaml`
-    - `go-tc-counter-install-vx.x.x.yaml`
-    - `go-tracepoint-counter-install-vx.x.x.yaml`
-    so that the examples for a given release can be easily deployed.
+  - run `make build-release-yamls` and attach the yamls for the version to the release. These will include:
```
Comment (Billy99) [inline RELEASE.md:104]:
```suggestion
  - run `VERSION=x.x.x make build-release-yamls` (where x.x.x is the patch version, i.e. 0.3.1) and attach the yamls for the version to the release. These will include:
```

File: RELEASE.md [hunk 2]
```diff
@@ -101,14 +101,12 @@ For a **PATCH** release:
...
+    - `bpfd-crds-install-v0.3.0.yaml`
+    - `bpfd-operator-install-v0.3.0.yaml`
+    - `go-xdp-counter-install-v0.3.0.yaml`
+    - `go-tc-counter-install-v0.3.0.yaml`
+    - `go-tracepoint-counter-install-v0.3.0.yaml`
```
Comment (Billy99) [inline RELEASE.md:109]: I think these should all be x.x.x like they were before.

File: RELEASE.md [hunk 3]
```diff
@@ -129,14 +127,12 @@ For a **MAJOR** or **MINOR** release:
...
 - The Release will be automatically created, after that is complete do the following:
-  - run `make build-crd-install-yaml` and attach the crd install yaml for the version to the release.
-  - manually build and attach a `bpfd-operator-install-vx.x.x.yaml` to provide an easy to use install
-    yaml for the bpfd-operator.
-  - manually build and attach the following files:
-    - `go-xdp-counter-install-vx.x.x.yaml`
-    - `go-tc-counter-install-vx.x.x.yaml`
-    - `go-tracepoint-counter-install-vx.x.x.yaml`
-    so that the examples for a given release can be easily deployed.
+  - run `make build-release-yamls` and attach the yamls for the version to the release. These will include:
```
Comment (Billy99) [inline RELEASE.md:130]: Same as above.

File: RELEASE.md (hunk 1 above)
Comment (Billy99) [reply RELEASE.md:104]: Ok, I was search for VERSION and didn't see it being set anywhere. I see it now.

File: RELEASE.md (hunk 3 above)
Comment (astoycos) [reply RELEASE.md:130]: Done

File: RELEASE.md (hunk 2 above)
Comment (astoycos) [reply RELEASE.md]: Done!

**Review by Billy99 — APPROVED** [review]
LGTM

**Pull request merged**